src/test/kotlin/uk/gov/dluhc/emsintegrationapi/rest/postal/DeletePostalVoteApplicationsIntegrationTest.kt

```diff
@@ -0,0 +1,173 @@
+import uk.gov.dluhc.emsintegrationapi.config.ApiClient
+import uk.gov.dluhc.emsintegrationapi.config.ApiProperties
+import uk.gov.dluhc.emsintegrationapi.config.ERO_CERTIFICATE_MAPPING_CACHE
+import uk.gov.dluhc.emsintegrationapi.config.ERO_GSS_CODE_BY_ERO_ID_CACHE
+import uk.gov.dluhc.emsintegrationapi.config.IntegrationTest
+import uk.gov.dluhc.emsintegrationapi.database.entity.RecordStatus
+import uk.gov.dluhc.emsintegrationapi.database.entity.SourceSystem
+import uk.gov.dluhc.emsintegrationapi.database.repository.PostalVoteApplicationRepository
+import uk.gov.dluhc.emsintegrationapi.messaging.models.EmsConfirmedReceiptMessage
+import uk.gov.dluhc.emsintegrationapi.testsupport.ClearDownUtils
+import uk.gov.dluhc.emsintegrationapi.testsupport.WiremockService
+import uk.gov.dluhc.emsintegrationapi.testsupport.testhelpers.PostalIntegrationTestHelpers
+import java.util.concurrent.TimeUnit
+
+class DeletePostalVoteApplicationsIntegrationTest : IntegrationTest() {
+    @Autowired
+    private lateinit var cacheManager: CacheManager
+
+    @Autowired
+    private lateinit var wireMockService: WiremockService
+
+    @Autowired
+    private lateinit var webClient: WebTestClient
+
+    @Autowired
+    private lateinit var apiProperties: ApiProperties
+
+    @Autowired
+    private lateinit var postalVoteApplicationRepository: PostalVoteApplicationRepository
+
+    @Autowired
+    private lateinit var queueMessagingTemplate: QueueMessagingTemplate
+
+    private var apiClient: ApiClient? = null
+
+    private var testHelpers: PostalIntegrationTestHelpers? = null
+
+    @BeforeEach
+    fun setup() {
+        cacheManager.getCache(ERO_CERTIFICATE_MAPPING_CACHE)?.clear()
+        cacheManager.getCache(ERO_GSS_CODE_BY_ERO_ID_CACHE)?.clear()
+        ClearDownUtils.clearDownRecords(postalRepository = postalVoteApplicationRepository)
+        apiClient = ApiClient(webClient, apiProperties)
+        testHelpers =
+            PostalIntegrationTestHelpers(
+                wiremockService = wireMockService,
+                postalVoteApplicationRepository = postalVoteApplicationRepository,
+                queueMessagingTemplate = queueMessagingTemplate,
+            )
+        testHelpers!!.givenEroIdAndGssCodesMapped()
+    }
+
+    @AfterEach
+    fun tearDown() {
+        cacheManager.getCache(ERO_CERTIFICATE_MAPPING_CACHE)?.clear()
+        cacheManager.getCache(ERO_GSS_CODE_BY_ERO_ID_CACHE)?.clear()
+        ClearDownUtils.clearDownRecords(postalRepository = postalVoteApplicationRepository)
+    }
+
+    @Test
+    fun `System returns http status 403 if certificate serial number is not attached to the request`() {
+        // When the EMS send a delete request to "/postalvotes" with an application id "502cf250036469154b4f85fb" and without the certificate serial number in the request header
+        val responseSpec = apiClient!!.delete("/postalvotes/502cf250036469154b4f85fb", attachSerialNumber = false)
```

Comment:
The 502cf250036469154b4f85fb comes up quite a lot in this file - is it worth extracting it out and adding it back in with string interpolation here? This happens in quite a few files and I don't want to bloat the review by commenting each of them, but the same applies to PostPostal and PutPostal, & similar for Proxy!